test/baseline/unit/launcher/aprun.txt

```diff
@@ -4,29 +4,29 @@ CASE: aprun
 INPUT: visit -engine -norun engine_par -l aprun -np 8 -nn 1 -host 127.0.0.1 -port 5600
 
 RESULTS:
-aprun -n 8 -N 8 $VISIT_EXE_DIR/engine_par -plugindir $HOME/.visit/$VERSION/$PLATFORM/plugins:$VISITDIR/plugins -visithome $VISITDIR -visitarchhome $VISITDIR/ -dv -host $HOST -port 5600
+aprun -n 8 -N 8 /usr/WS1/visit/test_trunk/visit/build/exe/engine_par -plugindir $HOME/.visit/$VERSION/$PLATFORM/plugins:/usr/WS1/visit/test_trunk/visit/build/plugins -visithome /usr/WS1/visit/test_trunk/visit/build -visitarchhome /usr/WS1/visit/test_trunk/visit/build/ -dv -host $HOST -port 5600
```

Comment:
Looks like these results use to be filtered so they could pass when run from a different directory. I think the filtering mechanism should be found and fixed for these tests.

Reply:
Take a look here:
https://github.com/visit-dav/visit/blob/a55f4a991335230fd5a5d419d2f493cc0699a7a2/src/test/tests/unit/launcher.py#L143-L152

Reply:
I was going to make some suggestions, but after updating my checkout of develop, this test isn't failing for me.